### PR TITLE
use constants instead of literals

### PR DESCRIPTION
**Pull request by chandiniv1**
closes: #349

### CODE REVIEW

**Comment by Manav-Aggarwal** [comment]
Thank you so much for submitting this! Looks great, just need to remove the changes to `go.mod` since we're still on go 1.20. Also, please check the CI tests pass after the update :)

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module github.com/rollkit/rollkit
 
-go 1.20
+go 1.21
+
+toolchain go1.21.1
```
Comment (Manav-Aggarwal) [inline go.mod:5]: We cannot make this upgrade yet since the current latest version of `ignite` does not support go 1.21. Please revert the changes in this file.

**Review by Manav-Aggarwal — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (chandiniv1) [reply go.mod]: @Manav-Aggarwal Done

**Review by Manav-Aggarwal — APPROVED** [review]

**Review by nashqueue — APPROVED** [review]
Thank you !
